pkg/cmd/run.go

```diff
@@ -582,6 +582,14 @@ func getPassphrase(cmd *cobra.Command, flag string, config models.ScopedOptions)
 		return cmd.Flag(flag).Value.String()
 	}
 
+	if configuration.CanReadEnv {
+		passphrase := os.Getenv("DOPPLER_PASSPHRASE")
+		if passphrase != "" {
+			utils.LogDebug("DOPPLER_PASSPHRASE setting passphrase")
```

Comment:
```suggestion
			utils.LogDebug("Setting passphrase via DOPPLER_PASSPHRASE env var")
```

Reply:
Just realized we have a `valueFromEnvironmentNotice` helper that you can use here

Reply:
Ah, perfect!